sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -64,6 +67,42 @@ class JoinSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  def assertJoinOrdering(sqlString: String, numOfJoin: Int, numOfSort: Int): Any = {
```

Comment:
nit: comparing the counts does not ensure that the sorts are in right place. I wish there was an easier way to pass that here but I can't think of any

Reply:
BTW: since this is only used by one test case, we could put it inside the test case method and not make it class level